.gitignore

```diff
@@ -47,3 +47,5 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+
+package-lock.json
```

Comment:
Please don't ignore the package-lock.json as it is needed for automated docker builds

Reply:
Also restore the delete file pls